fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownPredicateScanRule.java

```diff
@@ -112,4 +116,30 @@ public List<OptExpression> transform(OptExpression input, OptimizerContext conte
         OptExpression project = OptExpression.create(logicalProjectOperator, OptExpression.create(newScanOperator));
         return Lists.newArrayList(project);
     }
+
+    private ScalarOperator simplifyCoalesce(ScalarOperator predicate) {
```

Comment:
move to SimplifiedScanColumnRule?

Reply:
+1. we need to support simplify more exprs in the predicate.

Reply:
Ok

Reply:
this simplification cannot be performed indenpendently, for example: `where coalesce(v1, false) is NULL`, the coalesce cannot be removed directly